lua/bufferline/utils.lua

```diff
@@ -141,6 +161,18 @@ local utils = {
     return relative(path) == path
   end,
 
+  --- Reverse the order of elements in some `list`.
+  --- @generic T
+  --- @param list T[]
+  --- @return T[] reversed
+  list_reverse = function(list)
```

Comment:
There is `utils.list_slice_from_end`, and `vim.list_extend`, etc. But `reverse` was the only list function to not have the "list_" prefix.